packages/modal/src/Modal.js

```diff
@@ -222,14 +238,10 @@ export default class Modal extends React.Component<Props, State> {
                 {children}
               </div>
             )}
-            {(confirmText || denyText) && (
+            {(confirmText || denyText || confirmElm || denyElm) && (
```

Comment:
Gaat dit geen problemen geven als je bv geen denyText hebt maar wel een confirmText? Ga je dan geen lege button hebben?

Reply:
Klopt, zal een check toevoegen voor de trigger enkel te tonen als confirmText of confirmElm meegegeven wordt
confirmtext en denytext worden wel gedefault naar ja/nee, dus je zou al bewust `null` moeten meegeven om lege buttons te renderen

Reply:
Ideaal! Thanks :) 
Ja ik weet een paar scenarios waar we het zo doen in onze app